python/lighthouse/ingress/torch/torch_import.py

```diff
@@ -197,20 +187,40 @@ def get_inputs():
     if model is None:
         raise ValueError(f"Model class '{model_class_name}' not found in {filepath}")
 
-    if init_args_fn_name is None:
-        init_args_fn = lambda *args, **kwargs: ()
-    else:
-        init_args_fn = getattr(module, init_args_fn_name, None)
-        if init_args_fn is None:
-            raise ValueError(f"Init args function '{init_args_fn_name}' not found in {filepath}")
-
-    inputs_args_fn = getattr(module, inputs_args_fn_name, None)
-    if inputs_args_fn is None:
-        raise ValueError(f"Inputs args function '{inputs_args_fn_name}' not found in {filepath}")
+    model_init_args = maybe_load_and_run_callable(
```

Comment:
I believe the following works and would mean a bit less abstraction:
```python
try:
     model_init_args = init_args_fn_name and getattr(module, init_args_fn_name)() or tuple()
except AttributeError:
     raise ValueError(f"Init args function '{init_args_fn_name}' not found in {filepath}")
```
I know not everyone will find using the boolean operators in this way intuitive though (and technically it's not completely right, in the sense that returned falsity values will get replaced by `tuple()`). Nonetheless thought to suggest it.